### PR TITLE
Certificate Validation Check New KB Entries.

### PR DESCRIPTION
**Pull request by nmasdoufi-ol**
Certificate Validation Check New KB Entries.

### CODE REVIEW

File: WEB_SERVICE/WEB/_HIGH/CERTIFICATE_EXPIERD/description.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+An expired SSL/TLS certificate is no longer trusted by browsers and clients, leading to broken trust chains, security warnings, and exposure to potential security vulnerabilities, including man-in-the-middle attacks.
+
+Key Points:
+
+- Expired Certificates: Expired certificates are no longer trusted by browsers, causing alarming security warnings and broken trust, which can harm business operations.
+- User Impact: Security warnings deter users, often resulting in loss of trust and abandoned transactions, especially in e-commerce platforms.
+- Security Risks: Expired certificates can be exploited for man-in-the-middle attacks, exposing communications to interception or forgery.
```
Comment (3asm) [inline WEB_SERVICE/WEB/_HIGH/CERTIFICATE_EXPIERD/description.md:7]: I don't see the point of this.

File: WEB_SERVICE/WEB/_HIGH/CERTIFICATE_EXPIERD/recommendation.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+To address expired certificate issues:
+
+1. Implement automated monitoring tools to prevent certificates from expiring unnoticed.
```
Comment (3asm) [inline WEB_SERVICE/WEB/_HIGH/CERTIFICATE_EXPIERD/recommendation.md:3]: Would be nice to show examples. Even better to implement auto certificate provisioning, you can give example from kubernetes.

File: WEB_SERVICE/WEB/_HIGH/HOSTNAME_MISMATCH/description.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+A hostname mismatch in an SSL/TLS certificate occurs when the domain name specified in the certificate does not match the actual hostname of the server. This mismatch can lead to security warnings in browsers and potential vulnerabilities to man-in-the-middle attacks.
+
+Key points about hostname mismatches:
```
Comment (3asm) [inline WEB_SERVICE/WEB/_HIGH/HOSTNAME_MISMATCH/description.md:3]: Key point style is not typical to our KB.

**Review by 3asm — CHANGES_REQUESTED** [review]

File: WEB_SERVICE/WEB/_HIGH/CERTIFICATE_EXPIERD/recommendation.md (hunk 2 above)
Comment (nmasdoufi-ol) [reply WEB_SERVICE/WEB/_HIGH/CERTIFICATE_EXPIERD/recommendation.md]: Done.

File: WEB_SERVICE/WEB/_HIGH/HOSTNAME_MISMATCH/description.md (hunk 3 above)
Comment (nmasdoufi-ol) [reply WEB_SERVICE/WEB/_HIGH/HOSTNAME_MISMATCH/description.md]: Removed it.

**Review by 3asm — APPROVED** [review]

**Pull request merged**